src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -69,6 +70,31 @@ CaseDetails submitForCaseworker(
             @RequestParam Map<String, Object> searchCriteria
     );
 
+    @RequestMapping(
+            method = RequestMethod.GET,
+            value = "/caseworkers/{userId}/jurisdictions/{jurisdictionId}/case-types/{caseType}/cases/pagination_metadata",
+            headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE)
+    PaginatedSearchMetadata searchCasesMetadataForCaseworkers(@RequestHeader(AUTHORIZATION) String authorisation,
+                                                              @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorisation,
+                                                              @PathVariable String userId,
+                                                              @PathVariable String jurisdictionId,
+                                                              @PathVariable String caseType,
+                                                              @RequestParam Map<String, String> queryParameters
```

Comment:
For consistency, this should be called `searchCriteria`, same as for `searchForCaseworker()`.

Reply:
done